app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue

```diff
@@ -87,6 +87,7 @@
             track-by="name"
             :multiple="true"
             :taggable="true"
+            :preserve-search="true"
```

Comment:
What does the `preserve-search` do here? Doesn't seem like a valid prop for the element? As far as I know, there is no standard HTML attribute or property named `preserve-search` either

Reply:
Noted